### ISSUE TITLE
invalid hook call

### ISSUE THREAD

**Issue by jamalcoder** [meteor/react-packages#307]
after updating react-meteor-data from 2.0.1 to 2.1.3 i got this error
[
![invalid hook call](https://user-images.githubusercontent.com/11628272/101413269-3bafad80-38e4-11eb-890e-3ce85d317cf6.png)
](url)

**Comment by jasongrishkoff** [comment]
Getting this error on my side as well!

**Comment by rj-david** [comment]
Got the same error

```
Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.
    at resolveDispatcher (react-meteor-data.js?hash=085e11d486d61b046c1711ec1d5bf34be9d73759:1955)
    at useRef (react-meteor-data.js?hash=085e11d486d61b046c1711ec1d5bf34be9d73759:1994)
    at useTrackerNoDeps (useTracker.ts:140)
    at useTrackerClient (useTracker.ts:250)
    at useTrackerDev (useTracker.ts:281)
    at withTracker.tsx:17
    at renderWithHooks (modules.js?hash=324c5d688c2af2f867b7223d9464e6ff1436a4c1:25090)
    at updateForwardRef (modules.js?hash=324c5d688c2af2f867b7223d9464e6ff1436a4c1:27149)
    at beginWork (modules.js?hash=324c5d688c2af2f867b7223d9464e6ff1436a4c1:29203)
    at HTMLUnknownElement.callCallback (modules.js?hash=324c5d688c2af2f867b7223d9464e6ff1436a4c1:14050)
```

**Comment by renanccastro** [comment]
Fixed on 2.1.4, please update!
I published by mistake having the node_modules populated locally, so it went with another copy of React, sorry about that.
2.1.4 fixes it.

**Comment by rj-david** [comment]
Confirmng the fix
